MoltenVK/MoltenVK/Commands/MVKCmdPipeline.mm

```diff
@@ -346,6 +346,54 @@
 }
 
 
+#pragma mark -
+#pragma mark MVKCmdSetResetEvent
+
+void MVKCmdSetResetEvent::setContent(VkEvent event, VkPipelineStageFlags stageMask, bool status) {
+	_mvkEvent = (MVKEvent*)event;
+	_status = status;
+}
+
+void MVKCmdSetResetEvent::encode(MVKCommandEncoder* cmdEncoder) {
+	cmdEncoder->signalEvent(_mvkEvent, _status);
+}
+
+MVKCmdSetResetEvent::MVKCmdSetResetEvent(MVKCommandTypePool<MVKCmdSetResetEvent>* pool)
+	: MVKCommand::MVKCommand((MVKCommandTypePool<MVKCommand>*)pool) {}
+
+
+#pragma mark -
+#pragma mark MVKCmdWaitEvents
+
+void MVKCmdWaitEvents::setContent(uint32_t eventCount,
+							   const VkEvent* pEvents,
+							   VkPipelineStageFlags srcStageMask,
+							   VkPipelineStageFlags dstStageMask,
+							   uint32_t memoryBarrierCount,
+							   const VkMemoryBarrier* pMemoryBarriers,
```

Comment:
You don't seem to be doing anything with these.

Reply:
It wasn't clear what of those we could make use of in Metal's event environment. 

Do you have suggestions?

I was wondering if CTS would highlight differences in handling here...but so far all the CTS event tests pass.

Reply:
I was thinking of inserting a barrier, as `vkCmdPipelineBarrier()` does, either before or after the wait.

Reply:
Okay. I'll have a closer look at the barrier relationship. I'm not clear on the interaction either.